web/src/playground_links_popover.ts

```diff
@@ -95,6 +106,9 @@ function register_click_handlers() {
             // the url_template with the extracted code. Depending on whether
             // the language has multiple playground links configured, a popover
             // is shown.
+            if (playground_info === undefined) {
+                return;
+            }
```

Comment:
Why this change?

Reply:
Oops!, that's a mistake.` playground_info` throws when accessing `playground_info[0].url_template` . I think we should `assert` this. Thanks!.